### DISCUSSION TITLE
Hide labels/confidence

### DISCUSSION THREAD

**Discussion by aphilas** [obss/sahi#839] [Q&A] [answered]
How can I hide class labels and/or the confidence after calling `get_slice_prediction`?

I have checked the [docs](https://github.com/obss/sahi/tree/main/docs) and tried `model = yolov5.load(yolov5_model_path); model.hide_labels = True`.

**Comment by aphilas** [comment, marked as answer]
I modified `visualize_object_predictions` in [sahi/utils/cv.py#L400](https://github.com/obss/sahi/blob/906dd4d42730ff08e2e9b3adb3ef1e3e266dcfcb/sahi/utils/cv.py#L400).

**Reply by fcakyon** [reply]
Would you mind opening a PR?

**Reply by aphilas** [reply]
Alright, I'll give it a shot
